### PR TITLE
update REAME for recent changes

### PR DESCRIPTION
**Pull request by Jialn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,48 +47,44 @@ To see the graphics, you can open another terminal and run
...
+<img src="media/pioneer.jpg" width="160" height="240" alt="pioneer"/> <img src="media/pr2.jpg" width="160" height="240" alt="pr2"/> <img src="media/turtlebot.jpg" width="160" height="240" alt="turtlebot"/> <img src="media/icub.jpg" width="160" height="240" alt="icub"/> <img src="media/youbot.jpg" width="160" height="240" alt="youbot"/>
 
-    A simple navigation task to chase a goal. The agent will receive reward 1 when it is close enough to the goal, and get -1 it becomes too far away from the goal or timeout.
+The observation of the agent is also configuable, including internal states, target pose or camera image, and optional language instructions. The action specification is defined in the joint list of file [agent_cfg.json](python/social_bot/models/agent_cfg.json)
 
-    <img src="media/ac_simple_navigation.gif" width="320" height="240" alt="simple_navigation"/>
+* [Goal task](python/social_bot/teacher_tasks.py): A task to chase a goal. The agent will receive reward 1 when it is close enough to the goal, and get -1 it becomes too far away from the goal or timeout. A training example can be found [here](examples/goaltask_sac_pioneer.gin).
```
Comment (le-horizon) [inline README.md]: -> and get -1 "if" it

File: README.md [hunk 2]
```diff
@@ -43,48 +47,44 @@ To see the graphics, you can open another terminal and run
...
 
-### [Grocery Ground](python/social_bot/envs/play_ground.py)
-A playground with groceries on it. You could choose the agent in the environment by setting the paramenter "agent_type". We support pioneer2dx, pr2, turtlebot, kuka youbot, and icub for now.
-[ICub](http://www.icub.org) is an humanoid robot meant for more complex tasks in the future. You could also choose icub_with_hands, which is a more advanced version of icub equipped with 2 cameras and dexterous hands. The following are images of these 5 robots:
+    <img src="media/pioneer.gif" width="320" height="240" alt="pioneer"/> <img src="media/play_ground_icub.gif" width="360" height="240" alt="icub"/>
 
-<img src="media/pioneer.jpg" width="160" height="240" alt="pioneer"/> <img src="media/pr2.jpg" width="160" height="240" alt="pr2"/> <img src="media/turtlebot.jpg" width="160" height="240" alt="turtlebot"/> <img src="media/icub.jpg" width="160" height="240" alt="icub"/> <img src="media/youbot.jpg" width="160" height="240" alt="youbot"/>
+* [Kicking-ball task](python/social_bot/teacher_tasks.py): A task to kick a ball into the goal. Simple reward shaping is used to guide the agent run to the ball first.
```
Comment (le-horizon) [inline README.md]: guide the agent "to" run to ...

File: README.md [hunk 3]
```diff
@@ -99,18 +99,18 @@ The observation of the robots include internal states, target pose or camera ima
     <img src="media/icub_walk.gif" width="320" height="240" alt="pioneer"/>
 
 ### Training Examples with [Agent Learning Framework (Alf)](https://github.com/HorizonRobotics/alf)
-Train simple navigation task with [Alf actor-critic](examples/ac_simple_navigation.gin)
+Train goal_task task with [Alf actor-critic](examples/ac_simple_navigation.gin)
```
Comment (le-horizon) [inline README.md]: This results in an invalid link.  Maybe use absolute path?  https://github.com/HorizonRobotics/alf/blob/master/alf/examples/ac_simple_navigation.gin

File: README.md [hunk 4]
```diff
@@ -99,18 +99,18 @@ The observation of the robots include internal states, target pose or camera ima
...
 ```bash
-python -m alf.bin.play --root_dir=~/tmp/simple_navigation --gin_file=ac_simple_navigation.gin --num_episodes=20
+python -m alf.bin.play --root_dir=~/tmp/goal_task --gin_file=goaltask_sac_pioneer.gin --num_episodes=20
 ```
 The playing back can also be recorded to a video file like this:
 ```bash
-python -m alf.bin.play --root_dir=~/tmp/goal_task --gin_file=goaltask_alf_ppo.gin --record_file=goal_task.mp4
+python -m alf.bin.play --root_dir=~/tmp/goal_task --gin_file=goaltask_sac_pioneer.gin --record_file=goal_task.mp4
 ```
 You can find the gin config files for other tasks [here](examples).
```
Comment (le-horizon) [inline README.md]: https://github.com/HorizonRobotics/alf/tree/master/alf/examples

File: README.md (hunk 3 above)
Comment (Jialn) [reply README.md]: This is a local repo link, I've renamed ac_simple_navigation.gin and forgot to change the link here...

Comment (Jialn) [reply README.md]: Fixed now.

**Pull request merged**